cli/src/pcluster/cli/commands/cluster.py

```diff
@@ -45,9 +45,7 @@ def __init__(self, subparsers):
 
     def register_command_args(self, parser: ArgumentParser) -> None:  # noqa: D102
         parser.add_argument(
-            "cluster_name",
-            help="Defines the name of the cluster. The CloudFormation stack name will be "
-            f"{PCLUSTER_STACK_PREFIX}[cluster_name]",
+            "cluster_name", help="Defines the name of the cluster and the CloudFormation stack name will be "
```

Comment:
We can simplify it to: "Defines the name of the cluster"

Reply:
Done